### ISSUE TITLE
[Feature Request] Cron-style scheduling

### ISSUE THREAD

**Issue by soasme** [soasme/nim-schedules#6]
Request to support cron-style scheduling. For example:

```
import schedules, times, asyncdispatch

schedules:
  cron(expr="0 0 * * *", id="tick"):
    echo("tick", now())

  cron(expr="@every 1m", id="atick", async=true):
    echo("tick", now())
    await sleepAsync(3000)
```

**Comment by soasme** [comment]
Update: I'm working on this issue at the moment. It's a shame nim-lang doesn't have cron expression parser library so I have to write one first.

**Comment by soasme** [comment]
Close this issue since this feature has been released in v0.2.0.
Will keep improving it and ensuring it's stable.
